tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -11,7 +11,8 @@ Define {
 }
 
 Object.Dai {
-	ALH."514" {
+	ALH."1" {
+		dai_index 514
```

Comment:
hmm didn't have in kernel a macro for this "index translation" and then back in fw? if we use the indexes I guess we should use 0-16 etc. these feel random...

Reply:
this is not set in this PR @juimonen . Im simply moving it from the node ID to an attribute. The discussion about what it should be is not relevant for this PR. Maybe @bardliao can address that in a future PR

Reply:
ok

Reply:
@juimonen @ranj063 ALH dai_index is not used in IPC4 topology. We can keep it be 0 always.